media/js/firefox/australis/growth-firstrun-test1.js

```diff
@@ -0,0 +1,278 @@
+            }, 40000);
+        };
+
+        initFxA = function() {
+            csrf = randomStringMaker();
+
+            // set up FxA client
+            fxaClient = new window.FxaRelierClient(fxaRelier.clientId, {
+                contentHost: fxaRelier.contentHost,
+                oauthHost: fxaRelier.contentOauth
+            });
+
+            // conjure FxA lightbox
+            fxaClient.auth.signUp({
+                redirectUri: fxaRelier.redirectUri,
+                scope: 'profile',
+                state: csrf,
+                ui: 'lightbox',
+                background: '#fff',
+                zIndex: 99,
+                keepOverlay: true // customized build of fxa-relier-client.js to support this option
+            }).then(function(result) {
+                // make sure CSRF/state matches
+                if (result.state === csrf) {
+                    // change header
+                    $('#fxa-value-prop-intro').fadeOut('fast', function() {
+                        $('#fxa-value-prop-verified').fadeIn('fast');
+                    });
+
+                    // wipe out iframe and reposition delayed CTAs
+                    $fxaIframe.slideUp('normal', function() {
+                        $fxaIframe.remove();
+                    });
+
+                    $fxaDelayedCTAs.addClass('visible noiframe');
+                }
+            }, function(err) {
+                // on error (or click of X in iframe), get rid of overlay and fall back to default content
+                $fxaOverlay.remove();
+
+                // link directly to Firefox Accounts when clicking the Sync CTA button
+                Mozilla.UITour.getConfiguration('sync', function() {
+                    $('.sync-cta').on('click', '.button', trackSyncClick);
+                });
+
+                toggleDefaultContent(true);
+            });
+        };
+
+        // track Sync CTA click and link to about:accounts where posiible
+        trackSyncClick = function(e) {
+            e.preventDefault();
+
+            var goToAccounts = function () {
+                // available on Firefox 31 and greater
+                Mozilla.UITour.showFirefoxAccounts();
+            };
+
+            gaTrack(['_trackEvent', pageId + ' Page Interactions - New Firefox Tour', 'button click', 'Get Started with Sync'], goToAccounts);
+        };
+
+        randomStringMaker = function() {
+            var text = '';
+            var possible = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789';
+
+            for(var i = 0; i < 15; i++) {
+                text += possible.charAt(Math.floor(Math.random() * possible.length));
+            }
+
+            return text;
+        };
+
+        // listen for postMessage from iframes
+        window.addEventListener('message', function(event) {
+            // listen for fxa communiques
```

Comment:
I would like to, but I still don't know what the origin will be. I'm guessing it's the same as `fxaRelier.contentHost`, but I don't want to build that in without being sure. I think it's okay without the check for this test.